journal/modernized/05-chapter-3.adoc

```diff
@@ -297,16 +326,21 @@ Oh, that they may be a worthy people,
 to the praise and renown of the name of the Lord! is my souls cry to the God of my life,
 on their behalf.
 
+[.signed-section-signature]
 Charles Marshall.
 
+[.small-break]
+'''
+
 +++[+++The account which follows,
 of the peaceful close of this diligent labourer in the Lords vineyard,
-is extracted from Gough`'s History of Friends, and Piety Promoted: and as it is,
+is extracted from Gough`'s [.book-title]#History of Friends#, and [.book-title]#Piety Promoted:# and as it is,
 for the most part,
 additional to that contained in the Testimonies which precede the Journal,
-the Editor trusts it will not be considered unsuitably appended in this place.
+the Editor trusts it will not be considered unsuitably appended in this place.]
 
-* * *
+[.asterism]
```

Comment:
I didn't know if this asterisk should be left in between the intro to this account, or if it should be a [.small-break].